specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/compute.json

```diff
@@ -9040,7 +9040,7 @@
         },
         "licenseType": {
           "type": "string",
-          "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> If this element is included in a request for an update, the value must match the initial value. This value cannot be updated. <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/azure/virtual-machines/virtual-machines-windows-hybrid-use-benefit-licensing?toc=%2fazure%2fvirtual-machines%2fwindows%2ftoc.json) <br><br> Minimum api-version: 2015-06-15"
+          "description": "Specifies that the image or disk that is being used was licensed on-premises. This element is only used for images that contain the Windows Server operating system. <br><br> Possible values are: <br><br> Windows_Client <br><br> Windows_Server <br><br> For more information, see [Azure Hybrid Use Benefit for Windows Server](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/hybrid-use-benefit-licensing) <br><br> Minimum api-version: 2015-06-15"
```

Comment:
@rosingla needs to review this as we are now supporting AHUB for Linux VMs and we need to include the possible values for the license values support for Linux VMs and point to the reference for the more details.

Reply:
Suggested Description (same needs to be updated at all other places i.,e all other descriptions for licenseType field in this file) :

Specifies that the image or disk that is being used was licensed on-premises.

Possible values for Windows Server operating system are:

Windows_Client

Windows_Server

Possible values for Linux Server operating system are:

RHEL_BYOS

SLES_BYOS

For more information:

Azure Hybrid Use Benefit for Windows Server

Azure Hybrid Use Benefit For Linux Server

Minimum api-version: 2015-06-15

@smotwani / Abhishek/ Mayank to review and ack.

Reply:
ack

Reply:
acked by Mayank as well

Reply:
@rosingla , @smotwani , I made these suggested changes.